src/core/js/scrolling.js

```diff
@@ -56,7 +56,8 @@ define([
         const $app = Adapt.scrolling.$app;
         const $element = this;
         const elementOffset = selectorOffset.call($element);
-        const isCorrectedContainer = $element.parents().add($element).filter('html,body,#app').length;
+        const isCorrectedContainer = $element.is('html', 'body', '#app') ||
```

Comment:
Apologies; just noticed there are too many pesky apostrophes:
```suggestion
        const isCorrectedContainer = $element.is('html, body, #app') ||
```

Reply:
You'll have to do this as a separate pr. It's already merged and the branch is missing.

Reply:
https://github.com/adaptlearning/adapt_framework/pull/2877 nice, ta xx